ams/dashboard/package.json

```diff
@@ -1,68 +1,47 @@
 {
-  "name": "datalake-open",
-  "version": "0.1.0",
+  "name": "dashboard-new",
   "private": true,
+  "version": "0.0.0",
   "scripts": {
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "build:vue-dev": "vue-cli-service build --mode dev",
-    "build:vue-test": "vue-cli-service build --mode test",
-    "build:vue-prod": "vue-cli-service build --mode prod",
-    "test:unit": "vue-cli-service test:unit",
-    "lint": "vue-cli-service lint"
```

Comment:
Maybe we need a new lint command and need to change the pre-commit at the end of the file.